internal/webhook/external/events/event.go

```diff
@@ -0,0 +1,12 @@
+package events
+
+// Event contains the genericized read-only event information.
+type Event interface {
```

Comment:
I have a feeling that we won't need this.

See [this comment](https://github.com/akuity/kargo/pull/4000/files#r2076500407).

I think we're going to land in a place where we're basically muxing dynamically and the handlers that the request get routed to each know how to deal with their particular sort of `http.Request` -- meaning `http.Request` is, for all intents and purposes, already an "event interface."